applications/vanilla/modules/class.tagmodule.php

```diff
@@ -103,6 +103,15 @@ public function getData() {
 
         $this->autoContext();
 
+        $tagData = null;
+        $this->EventArguments['ParentID'] = $this->ParentID;
+        $this->EventArguments['tagData'] = &$tagData;
+        $this->fireEvent('getData');
+        if(is_array($tagData) && $tagData) {
```

Comment:
Coding standard nitpick: add a space after `if`.

Reply:
What if no tags are supposed to be displayed (e.g. the only tag is a "status" tag), so `$tagData` is an empty array (falsy)? In its current state, wouldn't the code continue on to the standard routine?